ExploreASL_ImportMaster.m

```diff
@@ -19,41 +19,77 @@
 
     %% Import Workflow
     
-    % We expect x.DataParPath to be the sourceStructure.json file, so the root directory should be the study directory
-	[x.StudyRoot,~,~] = xASL_fileparts(x.DataParPath);
+    % We expect x.opts.StudyRoot to be the sourceStructure.json file, so the root directory should be the study directory
```

Comment:
We discussed this here: https://github.com/ExploreASL/ExploreASL/issues/480

Reply:
But I think that's actually a bug (the x.DataParPath). I'll quickly have a look.

Reply:
So in theory we don't need this if statement at all, because it should have been checked in the initialize part already, but it kind of helps with redundancy.

Reply:
I improved the comment.